packages/x-components/src/store/__tests__/utils.ts

```diff
@@ -12,4 +13,5 @@ export type SafeStore<
   Getters extends Dictionary,
   Mutations extends MutationsDictionary<Mutations>,
   Actions extends ActionsDictionary<Actions>
-> = Omit<XActionContext<State, Getters, Mutations, Actions>, 'rootGetters' | 'rootState'>;
+> = Omit<Store<State>, 'dispatch' | 'commit' | 'state' | 'getters'> &
```

Comment:
Why are we using string literals instead of types as before?

Reply:
I don't got you. Those "strings" are actually members of Types